src/Checkbox.jsx

```diff
@@ -1,12 +1,19 @@
+// @flow
+
 import React, { Component } from 'react';
-import PropTypes from 'prop-types';
 
-class Checkbox extends Component {
-  constructor() {
-    super();
-    this.state = {
-      isChecked: false,
-    };
+type Props = {
+  label: string,
+  route: any,
```

Comment:
@EddyIonescu, I'm adding to @user512's comment: The ```route``` prop is not coming from a GraphQL endpoint. It is coming from the miniRoutes.geo.json file that @user512 linked to. I do like your idea of adding types to objects we get from the API via the GraphQL types, but I don't think that applies in this case, since the ```route``` values are not actually coming from the GraphQL endpoint.

Reply:
@user512 @utsab you're right! Sorry for confusing you. The route does currently come from the geojson, but I'm currently working on having it come from the API. In any case, we can leave it as `any` for now and change it once we've defined the type via the API.